jobs/delete_unsaved_candidates.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import datetime
+from skyportal.models import init_db, Candidate, DBSession
+from baselayer.app.env import load_env
+
+
+env, cfg = load_env()
+init_db(**cfg["database"])
+
+cutoff_datetime = datetime.datetime.now() - datetime.timedelta(
+    days=cfg["misc"]["days_to_keep_unsaved_candidates"]
```

Comment:
No problem here; just a reminder that `cfg["misc.days_to_keep_unsaved_candidates"]` is shorthand for the same.

Reply:
Done